src/main/kotlin/app/UserProfile.kt

```diff
@@ -0,0 +1,15 @@
+package app
+
+import org.eclipse.egit.github.core.User
+
+data class UserProfile(
+    val user: User,
+    val quarterCommitCount: Map<String, Int>,
+    val langRepoCount: Map<String, Int>,
+    val langStarCount: Map<String, Int>,
+    val langCommitCount: Map<String, Int>,
+    val repoCommitCount: Map<String, Int>,
+    val repoStarCount: Map<String, Int>,
+    val repoCommitCountDescriptions: Map<String, String?>,
+    val repoStarCountDescriptions: Map<String, String?>
```

Comment:
ensure the map implementation allows null values if you really intend for it to be `String?`

Reply:
thanks for confirming, this is sufficient for this PR 👍 (at a later date we can look closer at repoCommitCountDescriptions/repoStarCountDescriptions to see if the nullable value type is eneded)